packages/edit-site/src/components/editor/index.js

```diff
@@ -264,10 +268,9 @@ export default function Editor( { isLoading } ) {
 						}
 						sidebar={
 							isEditMode &&
-							isRightSidebarOpen && (
-								<>
-									<ComplementaryArea.Slot scope="core/edit-site" />
-								</>
+							isRightSidebarOpen &&
+							! isDistractionFree && (
```

Comment:
added to ensure the sidebar is not open by something else in DFM.

Reply:
yes